android/app/src/main/java/app/covidshield/module/ExposureCheckModule.kt

```diff
@@ -0,0 +1,108 @@
+package app.covidshield.module
+
+import android.util.Log
+import androidx.core.app.NotificationCompat
+import androidx.work.*
+import androidx.work.PeriodicWorkRequest.MIN_PERIODIC_FLEX_MILLIS
+import androidx.work.PeriodicWorkRequest.MIN_PERIODIC_INTERVAL_MILLIS
+import app.covidshield.extensions.parse
+import app.covidshield.extensions.toJson
+import app.covidshield.receiver.worker.ExposureCheckNotificationWorker
+import app.covidshield.receiver.worker.ExposureCheckSchedulerWorker
+import com.facebook.react.bridge.*
+import com.google.gson.annotations.SerializedName
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import java.util.concurrent.TimeUnit
+import kotlin.coroutines.CoroutineContext
+
+class ExposureCheckModule(private val context: ReactApplicationContext) : ReactContextBaseJavaModule(context), CoroutineScope {
+
+    override fun getName(): String = "ExposureCheck"
+
+    override val coroutineContext: CoroutineContext get() = Dispatchers.Default
+
+    private val workManager: WorkManager by lazy(LazyThreadSafetyMode.NONE) {
+        WorkManager.getInstance(context.applicationContext)
+    }
+
+    private val workerConstraints = Constraints.Builder()
+            .setRequiresCharging(false)
+            .setRequiresBatteryNotLow(false)
+            .build()
+
+    @ReactMethod
+    fun scheduleExposureCheck(data: ReadableMap, promise: Promise) {
```

Comment:
Add a comment as to why the 2 functions i.e. you don't want a visual notification to show.